Doc/library/cgitb.rst

```diff
@@ -54,6 +54,20 @@ analysis.
    value forces plain text output.  The default value is ``"html"``.
 
 
+.. function:: text(info[, context=5])
+
+   Handles the exception described by *info* (a 3-tuple containing the result of
+   :func:`sys.exc_info`), formatting its traceback as text and returning the
+   result as a string.
+
+
+.. function:: html(info[, context=5])
+
+   Handles the exception described by *info* (a 3-tuple containing the result of
+   :func:`sys.exc_info`), formatting its traceback as html and returning the
```

Comment:
It should be `HTML` (uppercase), as in the rest of the file.  It is lowercase only for the string argument.